tests/auto/perfdata/vector_static_clang/perf.data.expected.txt

```diff
@@ -1,1293 +1,1293 @@
-	0	/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/9.1.0/../../../../include/c++/9.1.0/bits/stl_iterator.h:522:-1
-	0	/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/9.1.0/../../../../include/c++/9.1.0/bits/stl_iterator.h:520:0	std::back_insert_iterator<std::vector<double, std::allocator<double> > >::operator=(double&&) vector_static_clang_v8.0.1 /tmp/vector_static_clang_v8.0.1 
-	0	/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/9.1.0/../../../../include/c++/9.1.0/bits/stl_algo.h:4450:-1
-	0	/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/9.1.0/../../../../include/c++/9.1.0/bits/stl_algo.h:4441:0	std::back_insert_iterator<std::vector<double, std::allocator<double> > > std::generate_n<std::back_insert_iterator<std::vector<double, std::allocator<double> > >, int, main::$_0>(std::back_insert_iterator<std::vector<double, std::allocator<double> > >, int, main::$_0) vector_static_clang_v8.0.1 /tmp/vector_static_clang_v8.0.1 
-	0	./vector.cpp:12:-1
-	402610	./vector.cpp:9:0	main vector_static_clang_v8.0.1 /tmp/vector_static_clang_v8.0.1 
-	4212df
-	420e50	__libc_start_main vector_static_clang_v8.0.1 /tmp/vector_static_clang_v8.0.1 
-	40250d
-	4024e0	_start vector_static_clang_v8.0.1 /tmp/vector_static_clang_v8.0.1 
+	ffffffffb8ae13f3 ffffffffb8ae13f3	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8ac5c28 ffffffffb8ac5c28	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8ad35ed ffffffffb8ad35ed	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb9600114 ffffffffb9600114	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8a93984 ffffffffb8a93984	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb9402ac6 ffffffffb9402ac6	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb9401caf ffffffffb9401caf	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8c91dcd ffffffffb8c91dcd	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8c9456d ffffffffb8c9456d	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8c946fd ffffffffb8c946fd	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8c3776c ffffffffb8c3776c	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8c37f8e ffffffffb8c37f8e	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8a75356 ffffffffb8a75356	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb8a75661 ffffffffb8a75661	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	ffffffffb94012ee ffffffffb94012ee	0 0
+	0 0	0 0	  [kernel.kallsyms]_text [kernel]
+	402710 402710	/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/9.1.0/../../../../include/c++/9.1.0/ext/new_allocator.h:147:4	0 0
+	402710 0	/usr/bin/../lib64/gcc/x86_64-pc-linux-gnu/9.1.0/../../../../include/c++/9.1.0/ext/new_allocator.h:144:0	402610 251	void __gnu_cxx::new_allocator<double>::construct<double, double>(double*, double&&) vector_static_clang_v8.0.1 /tmp/vector_static_clang_v8.0.1 
```

Comment:
I think for inlined frames we should still compute a relative address instead of 0, there's no reason not to do it. So this needs to be fixed in the previous patch which introduces this.

and this also shows why I would really like to see the patches themselves adding the test coverage - then it's clearly visible what they change and if they touch the tests at all

can you please do that? i.e. remove this patch altogether and instead in all of the previous patches directly write out the new params to add test coverage

Reply:
> I think for inlined frames we should still compute a relative address instead of 0, there's no reason not to do it. So this needs to be fixed in the previous patch which introduces this.

Do you mean pass relAddr into insertSubprogram instead 0 here:
int locationId = m_unwind->resolveLocation(PerfUnwind::Location(entry, 0, fileId, m_pid, line,
                                                                    column, inlineCallLocationId));
or something else?

Reply:
Yes, that. The 0 there is pretty arbitrary, no? We do output the absolute (non-relative) address too after all, so that's why I think the relative address should be there as well.

Reply:
Fixed